build-tools/packages/build-cli/src/releaseGroups.ts

```diff
@@ -33,7 +33,7 @@ export const knownReleaseGroups = [
  *
  * @internal
  */
-export type ReleaseGroup = typeof knownReleaseGroups[number] | string;
+export type ReleaseGroup = (typeof knownReleaseGroups)[number] | string;
```

Comment:
@tylerbutler - Just FYI - This evaluates to 'type ReleaseGroup = string'.

(typeof someArray)[number] extracts the T from Array<T>.  Since all the array elements are strings, you get:

= string | string
= string

For reference:
```ts
export const knownReleaseGroups: readonly ["build-tools", "client", "server", "gitrest", "historian"];
```

Reply:
Thanks, known issue. The problem is that there is some old code that uses string enums to represent release group names, and so the ReleaseGroup type currently serves as an interop layer of sorts (e.g. the "monorepo" class has a kind property. For fun, you can remove the `| string` and see the compile errors pretty easily. Very open to ideas about better approaches.

Ultimately what I want is to have APIs that are typed to a string literal union, but allow `string`s to be passed with minimal fuss. It's mostly safe because eventually any code that is looking up a release group will try to find something on disk, so even if a "bad string" makes it through the type checking, there will be a clear runtime failure.

That said maybe I should just get more comfortable with `as ReleaseGroup`?

Reply:
A better way to day it might be that I want to have APIs that are typed to strings, but I also want the compiler to prevent me from passing known bad strings at compile time whenever possible. In this case it's less critical that every bad string be caught at compile time - but if it can catch _some_ it's a win.

Reply:
NP.  Was just checking to make sure it was intentional.  Thx.